blog/modules/ROOT/examples/sort-components/componentSorter.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+module.exports = (components, sortOrder = undefined) => {
+  if (sortOrder) {
+    components = Object.assign({}, components)
+    sortOrder = sortOrder.split(',').map((componentName) => componentName.trim())
```

Comment:
If `sortOrder` is not specified, wouldn't `.split()` throw an error? Perhaps the default value for `sortOrder` should be an empty string.

Reply:
Well, if the second parameter is left out of the .hbs, you get what I think is the UI model as the second parameter and split definitely doesn't work there :-).
I checked that if the key is missing, you get 'undefined' and the `if (sortOrder) ...` check keeps things from blowing up.
So, I think the default for sortOrder isn't actually useful, but the code seems to work OK.

Reply:
Ah, sorry; missed the `if` statement. Have to remind myself not to upgrade from "skim" to "review" while working on other code. 😄